src/calibrate-multi-hop/.gitignore

```diff
@@ -0,0 +1,10 @@
+.env
+__pycache__/
+env/
+__init__test.py
+.gitkeep
+.vscode
+.DS_Store
+airqo-250220-5149c2aac8f2.json
```

Comment:
Hi @bobturneruk 
I don't seem to have this json file, the one shared in our channel by @Priscilla-AL has this name: **AirQo-e37846081569.json**

Reply:
I don't have a local json file at all for this.

Reply:
Is one needed for this microservice, for credentials?